.docksal/commands/site-build

```diff
@@ -26,6 +26,7 @@ echo "Running Drush Cache-Rebuild"
 drush cr
 
 echo "Running Gulp Tasks on theme"
```

Comment:
Good point. This should get removed. Also, they get ran in both in the event someone just runs the `site-build` without running `init`. But in theory we should modify this to check for the theme directory.

Reply:
@sean-e-dietrich I've pushed some changes. Think that should take things in line. I changed `gulp sass` to `gulp` to run the default task instead of just `sass` since there might be other tasks that should all be run at the same time.